### PR TITLE
Restclient refactor (#3731)

### PR DESCRIPTION
**Pull request by Andrii-Yezenitskyi**
RestClient split into 4 clients/services.

### CODE REVIEW

**Review by SomeOne35719 — APPROVED** [review]

File: service-api/src/main/java/greencity/client/FondyClient.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+/**
+ * Client for interacting with Fondy API.
+ *
+ * @author Andrii Yezenitskyi
+ */
+@FeignClient(name = "fondy-client",
+    url = "https://pay.fondy.eu/api")
```
Comment (thesvider) [inline service-api/src/main/java/greencity/client/FondyClient.java]: url should be defined as config property

File: service-api/src/main/java/greencity/client/ViberClient.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+
+/**
+ * Client for interacting with Viber API.
+ *
+ * @author Andrii Yezenitskyi
+ */
+@FeignClient(name = "viber-client",
+    url = "https://chatapi.viber.com",
```
Comment (thesvider) [inline service-api/src/main/java/greencity/client/ViberClient.java]: url should be defined as config property

File: service-api/src/main/java/greencity/client/ViberClient.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    ResponseEntity<String> updateWebHook(WebhookDto dto);
+
+    /**
+     * Sends a message to user.
+     *
+     * @param message {@link SendMessageToUserDto}
+     *
+     * @return {@link String} - raw JSON with status and additional details.
+     */
+    @PostMapping("/pa/send_message")
```
Comment (thesvider) [inline service-api/src/main/java/greencity/client/ViberClient.java]: since /pa/ is in each method, you can define it as a part base url

**Review by thesvider — COMMENTED** [review]
please resolve minor comments

**Review by Pavl0-Gural — APPROVED** [review]

**Review by yourromchikkkkk — APPROVED** [review]

**Pull request merged**